openedx/features/edly/validators.py

```diff
@@ -0,0 +1,53 @@
+from logging import getLogger
+
+from django.db import DataError
+from django.db.models import Q
+
+from openedx.features.edly.models import (
+    EdlyUserProfile,
+    EdlySubOrganization
+)
+
+logger = getLogger(__name__)
+
+
+def validate_user_access_with_current_edly_sub_organization(request, possibly_authenticated_user):
+    """
+    Validate possibly authenticated user's access based on request site
+    and its linked edly sub organization.
+
+    Arguments:
+        request: HTTP request object
+        possibly_authenticated_user: User object trying to authenticate
+
+    Returns:
+        bool: Returns True if User has Edly Organization Access Otherwise False.
+    """
+
+    current_site = request.site
+
+    try:
+        current_site_edly_sub_org = EdlySubOrganization.objects.get(
```

Comment:
why aren't we using `request.site.edly_sub_org_for_lms` here?

Reply:
Because I wasn't aware, we are capable of doing that too.

Reply:
Kindly use `request.site.edly_sub_org_for_lms`.